locale/en.po

```diff
@@ -1726,6 +1726,75 @@ msgstr ""
 "version.\n"
 "Please load the save manually through the menu."
 
+#: ../src/gui_common/TweakedColourPicker.cs:211
+msgid "COLOUR_PICKER_PICK_COLOUR"
+msgstr "Pick a colour from window"
+
+#: ../src/gui_common/TweakedColourPicker.cs:212
+msgid "COLOUR_PICKER_ADD_PRESET"
+msgstr "Add current colour as a preset"
+
+#: ../src/gui_common/TweakedColourPicker.cs:213
+msgid "COLOUR_PICKER_HSV_BUTTON_TOOLTIP"
+msgstr ""
+"Switch HSV (Hue, Saturation, Value) mode on or off.\n"
+"Can't be turned on when in raw mode."
+
+#: ../src/gui_common/TweakedColourPicker.cs:214
+msgid "COLOUR_PICKER_RAW_BUTTON_TOOLTIP"
+msgstr ""
+"Switch raw mode on or off.\n"
+"In raw mode you can make the R, G, B\n"
+"colour values to go beyond 1.0.\n"
+"Can't be turned on in HSV mode."
+
+#: ../src/gui_common/TweakedColourPicker.cs:218
+msgid "COLOUR_PICKER_H_TOOLTIP"
+msgstr "Hue value, portion of colour"
+
+#: ../src/gui_common/TweakedColourPicker.cs:219
+msgid "COLOUR_PICKER_S_TOOLTIP"
+msgstr "Saturation value, the amount of gray in a particular colour"
+
+#: ../src/gui_common/TweakedColourPicker.cs:220
+msgid "COLOUR_PICKER_V_TOOLTIP"
+msgstr "Value (brightness) value, brightness or intensity of the colour"
+
+#: ../src/gui_common/TweakedColourPicker.cs:224
+msgid "COLOUR_PICKER_R_TOOLTIP"
+msgstr "Red channel value of the colour"
+
+#: ../src/gui_common/TweakedColourPicker.cs:225
+msgid "COLOUR_PICKER_G_TOOLTIP"
+msgstr "Green channel value of the colour"
+
+#: ../src/gui_common/TweakedColourPicker.cs:226
+msgid "COLOUR_PICKER_B_TOOLTIP"
+msgstr "Blue channel value of the colour"
+
+#: ../src/gui_common/TweakedColourPicker.cs:229
+msgid "COLOUR_PICKER_A_TOOLTIP"
+msgstr "Alpha value of the colour"
+
+#: ../src/gui_common/TweakedColourPicker.cs:342
+msgid "COLOUR_PICKER_PRESET_TOOLTIP"
+msgstr ""
+"Color: #{0}\n"
+"Left mouse: Use this preset\n"
+"Right mouse: Delete this preset"
+
+#: ../src/gui_common/TweakedColourPicker.tscn:38
+msgid "HSV"
+msgstr "HSV"
+
+#: ../src/gui_common/TweakedColourPicker.tscn:47
+msgid "RAW"
+msgstr "Raw"
+
+#: ../src/gui_common/TweakedColourPicker.tscn:54
+msgid "#"
+msgstr "#"
```

Comment:
Could be wrong but I don't think this is necessary be translated.

Reply:
I don't want it to be translated, but update_localization.rb does. I don't know how to tell it to ignore that.

Reply:
Put `PLACEHOLDER` in the node's editor description so the translation system ignores it. See [documentation](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/working_with_translations.md#working-in-scene-files).

Reply:
Okay. I misunderstood that for setting the string displayed as "PLACEHOLDER" earlier.

Reply:
Okay. I misunderstood that for setting the string displayed as "PLACEHOLDER" earlier.